shared/src/main/scala/VNum.scala

```diff
@@ -46,14 +46,17 @@ object VNum:
     // Spits into real and imaginary parts
     // todo handle empty real part
     val parts = s.split("ı")
-    val real = parts(0)
-    val imag = parts.lift(1).getOrElse("0")
+    val real = parts.lift(0).getOrElse("0")
+    val imag = if s.endsWith("ı") then "1" else parts.lift(1).getOrElse("0")
```

Comment:
Not necessary but there's [this](https://scastie.scala-lang.org/XjPy7l5sSLi7KQac0naayA) cool feature that lets you match on strings